property-service/src/main/java/org/egov/ps/service/EnrichmentService.java

```diff
@@ -315,13 +317,22 @@ public void processNotification(List<Map<String, Object>> notificationConfigs, A
 			List<Notifications> notificationList = mapper.convertValue(notificationConfigs, new TypeReference<List<Notifications>>() { });
 
 			notificationList.forEach(data -> {
+				
+
+				Pattern p = Pattern.compile("\\{(.*?)\\}");
+				Matcher m = p.matcher(data.getContent());
+				List<String> allMatches = new ArrayList<String>();
+				while(m.find()) {
+				    allMatches.add(m.group());
+				}
+				
 				data.setContent(data.getContent()
 						//app.getModuleType() == null ? "" : app.getModuleType()
-						.replace("{createdBy.name}",enrichedApplication.getAuditDetails() == null ? "" : enrichedApplication.getAuditDetails().getCreatedBy())
-						.replace("{moduleType}", enrichedApplication.getModuleType() == null ? "" : enrichedApplication.getModuleType())
-						.replace("{applicationType}", enrichedApplication.getApplicationType() == null ? "" : enrichedApplication.getApplicationType())
-						.replace("{property.fileNumber}", enrichedApplication.getProperty() == null ? "" : enrichedApplication.getProperty().getFileNumber())
-						.replace("{applicationNumber}", enrichedApplication.getApplicationNumber() == null ? "" : enrichedApplication.getApplicationNumber())
+						.replace(allMatches.get(0),enrichedApplication.getAuditDetails() == null ? "" : enrichedApplication.getAuditDetails().getCreatedBy())
```

Comment:
Where are we using JSONPath? 
We have to loop for `allMatches` and use JSONPath to extract the value from `enrichedApplication` object.